sea-orm-macros/Cargo.toml

```diff
@@ -25,3 +25,6 @@ proc-macro2 = "^1"
 [dev-dependencies]
 sea-orm = { path = "../", features = ["macros"] }
 serde = { version = "^1.0", features = ["derive"] }
+
+[features]
+with-table-iden = []
```

Comment:
I don't think it needs to be a feature in `sea-orm-macros`. It should be exposed as an additional attribute to the macro.

I could not find any proposed interface, but I imagine it could be:

```rust
#[derive(Clone, Debug, PartialEq, DeriveEntityModel)]
#[sea_orm(table_name = "cake", with_table_iden)]
pub struct Model { ... }
```

Reply:
Same here? Don't want it to be feature gated?

Reply:
Ok will work on that!